imagick/imagick.go

```diff
@@ -345,6 +368,17 @@ func (i *Image) sizeFrames(sz *imgry.Sizing) error {
 }
 
 func (i *Image) WriteToFile(fn string) error {
+	if i.convertFormat != i.format {
+		if i.convertFormat == "mp4" {
```

Comment:
This is an special case, we may be able to support different destination files, but for now we only expect `mp4`.